batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentController.java

```diff
@@ -122,21 +124,25 @@ private void edit(HttpServletRequest request, HttpServletResponse response, int
 
   private void editProcess(HttpServletRequest request, HttpServletResponse response, int id)
     throws ServletException, IOException, DataException {
-    String name = request.getParameter("name");
-    String address = request.getParameter("address");
+
+    Map<String,String> studentMap = createHashMapFromInputs(request);
+    studentMap.put("id",String.valueOf(id));
     Student student = new Student();
+
     try {
-      int roll = Integer.parseInt(request.getParameter("roll"));
-      student.setName(name);
-      student.setAddress(address);
-      student.setRoll(roll);
+      StudentFactory studentFactory = new StudentFactory();
+      student = studentFactory.createObject(studentMap);
       student.setId(id);
 
+      request.setAttribute("student", studentMap);
       studentService.update(student);
       response.sendRedirect(request.getContextPath() + "/" + CommonConstants.LIST_URL);
-    } catch (NumberFormatException e) {
-      request.setAttribute("error", "invalid roll");
-      request.setAttribute("student", studentService.fetchById(id));
+    } catch (ValidatorException e) {
+      LOGGER.log(Level.SEVERE, e.getMessage(), e);
+      request.setAttribute("error", e.getErrors());
+
+//      request.setAttribute("student", studentService.fetchById(id));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)